torrt/trackers/anilibria.py

```diff
@@ -6,7 +6,7 @@
 
 LOGGER = logging.getLogger(__name__)
 
-REGEX_QUALITY = re.compile(ur".+\[(.+)\]")
+REGEX_QUALITY = re.compile(r".+\[(.+)\]")
```

Comment:
We might need unicode_literals import for that case.

Reply:
Is there any real need for that unicode flag here? AFAIK, none of the characters used in regex template are somehow affected by it.

Reply:
Nope, but it sould be helpful for py3 transition.
Anyway, as without a proper testcase this is next to useless, we may add it when a time comes.